types/relay-runtime/lib/store/RelayModernEnvironment.d.ts

```diff
@@ -25,43 +24,51 @@ import { RelayObservable } from '../network/RelayObservable';
 export interface EnvironmentConfig {
     readonly configName?: string;
     readonly handlerProvider?: HandlerProvider | null;
+    readonly treatMissingFieldsAsNull?: boolean;
     readonly log?: LogFunction | null;
     readonly operationLoader?: OperationLoader | null;
     readonly network: Network;
     readonly scheduler?: TaskScheduler | null;
     readonly store: Store;
     readonly missingFieldHandlers?: ReadonlyArray<MissingFieldHandler> | null;
     readonly operationTracker?: OperationTracker | null;
+    readonly options?: any;
```

Comment:
Q: Is `unknown` better than `any`?

Reply:
I assume the default is `undefined`, which is what I used in my generic example. If we don’t go down that route, through, I agree that `unknown` might be the better choice.